Packs/CommonTypes/ReleaseNotes/3_2_1.md

```diff
@@ -0,0 +1,98 @@
+
+#### Incident Fields
```

Comment:
Notice some fields were renamed in my recent commits

Reply:
Tnx - running RN again. Note that for the "Alert Category" field - I can't remove the entry from RN since we made changes on this field.
![image](https://user-images.githubusercontent.com/88268646/150743637-43c9891d-8d31-4ad1-90e7-79ecc2f3135b.png)